templates/intro-jobs/README.md

```diff
@@ -58,10 +61,29 @@ You should see the job state and its output on the overview page.
 
 <img src="https://raw.githubusercontent.com/anyscale/templates/main/templates/intro-jobs/assets/anyscale-job.png" height=400px>
 
+## Submitting a Job programmatically
+
+In the above section, you submitted a Job from a workspace. By default, Jobs submitted from workspaces inherit the dependencies and compute config of the workspace.
+
+You can also submit jobs from other machines, using the Anyscale CLI. 
+
+Copy ``main.py`` to an empty folder on your **laptop**, and then run the following on your **laptop** to try this out:
+
+```bash
+# Make sure we have anyscale CLI installed.
+$ pip install -U anyscale
+
+# Note: outside of workspaces, you must specify required files via --working-dir.
+$ anyscale job submit --working-dir=. --wait -- python main.py
+```
+
+Jobs submitted externally will run with the Anyscale default compute config and dependencies. To override these settings, use the ``--config-file``, ``--image-uri``, or ``--containerfile`` flags.
```

Comment:
Btw, @edoakes is the default the serverless config?

Reply:
yes it is